lib/rules/placeholder-name-format.js

```diff
@@ -18,60 +18,60 @@ module.exports = {
           strippedName,
           violationMessage = false;
 
-      strippedName = name;
-
-      if (parser.options['allow-leading-underscore'] && name[0] === '_') {
-        strippedName = name.slice(1);
-      }
-
-      switch (parser.options.convention) {
-      case 'hyphenatedlowercase':
-        if (!helpers.isHyphenatedLowercase(strippedName)) {
-          violationMessage = 'Placeholder \'%' + name + '\' should be written in lowercase with hyphens';
-        }
-        break;
-      case 'camelcase':
-        if (!helpers.isCamelCase(strippedName)) {
-          violationMessage = 'Placeholder \'%' + name + '\' should be written in camelCase';
+      if (node.first().is('ident')) {
+        strippedName = name;
+        if (parser.options['allow-leading-underscore'] && name[0] === '_') {
+          strippedName = name.slice(1);
         }
-        break;
-      case 'snakecase':
-        if (!helpers.isSnakeCase(strippedName)) {
-          violationMessage = 'Placeholder \'%' + name + '\' should be written in snake_case';
-        }
-        break;
-      case 'strictbem':
-        if (!helpers.isStrictBEM(strippedName)) {
-          violationMessage = 'Placeholder \'.' + name + '\' should be written in BEM (Block Element Modifier) format';
-        }
-        break;
-      case 'hyphenatedbem':
-        if (!helpers.isHyphenatedBEM(strippedName)) {
-          violationMessage = 'Placeholder \'.' + name + '\' should be written in hyphenated BEM (Block Element Modifier) format';
-        }
-        break;
-      default:
-        if (!(new RegExp(parser.options.convention).test(strippedName))) {
-          violationMessage = 'Placeholder \'%' + name + '\' should match regular expression /' + parser.options.convention + '/';
 
-          // convention-message overrides violationMessage
-          if (parser.options['convention-explanation']) {
-            violationMessage = parser.options['convention-explanation'];
+        switch (parser.options.convention) {
+        case 'hyphenatedlowercase':
```

Comment:
Nope, It's how eslint is set.. Changing to how we would write it gives indentation warning, also travis is already doing the linting tests & you'll wanna do the PR with ?w=1 as i only added 2 lines..